src/aws/auth.rs

```diff
@@ -4,6 +4,10 @@ use aws_config::{
 use aws_types::{credentials::SharedCredentialsProvider, region::Region, Credentials};
 use serde::{Deserialize, Serialize};
 
+// matches default load timeout from the SDK as of 0.10.1, but lets us confidently document the
+// default rather than relying on the SDK default to not change
+const DEFAULT_LOAD_TIMEOUT_SECS: u64 = 5;
```

Comment:
This is only used as a `Duration`, so maybe just make the constant also a `Duration`? (Then you don't have to name it "_secs")